finders/gps.py

```diff
@@ -1,20 +1,22 @@
 from dataclasses import dataclass
```

Comment:
If you want to conform to PEP8
```suggestion
"""
Module for GPS data
"""
from dataclasses import dataclass
```